cli/command/volume/cmd.go

```diff
@@ -36,13 +36,13 @@ volume is a specially-designated directory that by-passes storage driver
 management.
 
 Data volumes persist data independent of a container's life cycle. When you
-delete a container, the Engine daemon does not delete any data volumes. You can
+delete a container, the Docker daemon does not delete any data volumes. You can
 share volumes across multiple containers. Moreover, you can share data volumes
 with other computing resources in your system.
 
 To see help for a subcommand, use:
 
-    docker volume CMD help
+    docker volume COMMAND --help
```

Comment:
I can not find it which is a long  description for CLI command.  I modified it  which referring to the help as follows: 
$ docker volume --help

Usage:	docker volume COMMAND

Manage Docker volumes

Options:
      --help   Print usage

Commands:
  create      Create a volume
  inspect     Display detailed information on one or more volumes
  ls          List volumes
  rm          Remove one or more volumes

Run 'docker volume COMMAND --help' for more information on a command.